dist/tools/compile_and_test_for_board/compile_and_test_for_board.py

```diff
@@ -273,7 +277,10 @@ def run_compilation_and_test(self, **test_kwargs):
             return (str(err), err.application.appdir, err.errorfile)
 
     def compilation_and_test(self, clean_after=False, runtest=True,
-                             incremental=False, jobs=False):
+                             incremental=False, jobs=False,
+                             with_test_only=False):
+        # It reached too many arguments but not the good time for changing API
```

Comment:
Why isn't it a good time?

Reply:
It would require refactoring the way it is implemented and an internal API change, that I would prefer doing for a bigger change than just adding one option. Like when adding `TEST_ON_CI_BLACKLIST` handling.

I could remove the comment if you prefer, because it is more a comment for here than the code.

Reply:
```suggestion
```
I would just remove the comment. But feel free to ignore.

Reply:
I removed it